.github/workflows/testing-develop.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
     - develop
+    - GSOC2022-JatinJain
   pull_request:
     branches:
     - develop
+    - GSOC2022-JatinJain
```

Comment:
muss das nicht raus?

Reply:
yes, in a further step. There is also some wording in the docs, I don't wanted to change more than flake8 in the merge. There will be follow up PR.